src/worker-type.js

```diff
@@ -466,7 +466,8 @@ WorkerType.createLaunchSpec = function(bid, worker, keyPrefix, provisionerId, pr
   // Find the AZ object, if any
   let selectedAZ = {};
   let foundAZ = false;
-  for (let r of worker.AZs || []) {
+  let availabilityZones = worker.availabilityZones || [];
+  for (let r of availabilityZones) {
```

Comment:
I know it's not new in this patch, but I was thinking that instead of r, maybe az would be a little more clear to read?